src/maptm/src/maptm_ovsdb.c

```diff
@@ -0,0 +1,307 @@
+/*
+* Copyright (c) 2020, Sagemcom.
+* All rights reserved.
+*
+* Redistribution and use in source and binary forms, with or without
+* modification, are permitted provided that the following conditions are met:
+*
+* 1. Redistributions of source code must retain the above copyright notice,
+*    this list of conditions and the following disclaimer.
+*
+* 2. Redistributions in binary form must reproduce the above copyright notice,
+*    this list of conditions and the following disclaimer in the documentation
+*    and/or other materials provided with the distribution.
+*
+* 3. Neither the name of the copyright holder nor the names of its contributors
+*    may be used to endorse or promote products derived from this software
+*    without specific prior written permission.
+*
+* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+* AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+* IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+* ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+* LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+* CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+* SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+* INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+* CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+* ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+* POSSIBILITY OF SUCH DAMAGE.
+*/
+
+#include <stdio.h>
+
+#include "ovsdb_update.h"
+#include "ovsdb_sync.h"
+#include "ovsdb_table.h"
+#include "schema.h"
+
+#include "maptm.h"
+
+#ifdef MAPTM_DEBUG
+#undef LOGI
+#define LOGI    printf
+#endif
+
+/**
+ * Globals
+ */
+
+/* Log entries from this file will contain "OVSDB" */
+#define MODULE_ID LOG_MODULE_ID_OVSDB
+#define MAPTM_IFC_WAN "br-wan"
+
+ovsdb_table_t table_Netfilter;
+ovsdb_table_t table_Node_Config;
+struct ovsdb_table table_Node_State;
+static ovsdb_table_t table_Interface;
+static ovsdb_table_t  table_IP_Interface;
+ovsdb_table_t  table_Wifi_Inet_Config;
+ovsdb_table_t  table_DHCPv6_Client;
+ovsdb_table_t  table_DHCP_Client;
+ovsdb_table_t  table_IPv6_Address;
+
+/* To change to Enum  */
+int WanConfig = 0;
+
+// Testing MAP-T tables presence
+bool maptm_ovsdb_tabs_ready()
+{
+    json_t *where_config = NULL;
+    bool retval = false;
+    
+    //Checking MAP-T entry presence in Node_Config table
+    where_config = ovsdb_where_multi(
+            ovsdb_where_simple_typed(SCHEMA_COLUMN(Node_Config, module),"WANO", OCLM_STR),
```

Comment:
```suggestion
            ovsdb_where_simple_typed(SCHEMA_COLUMN(Node_Config, module), "WANO", OCLM_STR),
```
+ It should probably be "MAPTM", not "WANO"